components/navigation/SubCategoryDocsNav.js

```diff
@@ -0,0 +1,34 @@
+import { useState, useEffect } from "react";
+import DocsNavItem from "./DocsNavItem";
+import DocsArrow from "../icons/DocsArrow";
+
+export default function SubCategoryDocsNav({ subCategory, active, onClick }) {
```

Comment:
we  can also cange this `active` to something self explanatory such as `activeItem`